### DISCUSSION TITLE
Improve incoming-packet-threshold documentation

### DISCUSSION THREAD

**Discussion by ghost** [PaperMC/Paper#11295] [Ideas]
### Is your feature request related to a problem?

"Sets the threshold at which the server will consider incoming packets spam and ignore them."

This is incorrect, its only for USE_ITEM and USE_ITEM_ON packets

### Describe the solution you'd like.

clarify its only for right click interaction packets.

### Describe alternatives you've considered.

N/A

**Comment by electronicboy** [comment]
For issues with documentation, please create an issue on the docs repo
